### DISCUSSION TITLE
what's the entity's name in android ?

### DISCUSSION THREAD

**Discussion by wingdi** [google/filament#4801] [Q&A] [answered]
About entity's info , i find this in FilamentAsset.class:
```
This class owns a hierarchy of entities that have been loaded from a glTF asset. Every entity has a TransformManager component, and some entities also have NameComponentManager and/or RenderableManager components.
```
I thought its name is glTF's Json Object's Key,  but it's not ,  i use filametAsset.getEntityByName("meshes") got nothing,  any info I can know about this?  @romainguy

**Comment by wingdi** [comment, marked as answer]
solved.  it's node's name

**Reply by mbalajee** [reply]
Except that there is no Node in filament
